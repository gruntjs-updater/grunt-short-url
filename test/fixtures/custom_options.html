<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Custom options</title>
	<link rel="stylesheet" href="http://yui.yahooapis.com/pure/0.5.0/pure-min.css">
</head>
<body>
	<img src="http://lorempicsum.com/simpsons/627/300/4" alt="">

	<a href="https://github.com/mikaelguillin/grunt-short-url">github.com/mikaelguillin/grunt-short-url</a>

	<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
</body>
</html>