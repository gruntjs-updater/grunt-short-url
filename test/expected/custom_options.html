<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Custom options</title>
	<link rel="stylesheet" href="http://yui.yahooapis.com/pure/0.5.0/pure-min.css">
</head>
<body>
	<img src="http://goo.gl/5P0K0K" alt="">

	<a href="http://goo.gl/rU0VuC">github.com/mikaelguillin/grunt-short-url</a>

	<script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
</body>
</html>